packages/widget-utils/src/bootstrap/bootstrapWidgets.js

```diff
@@ -60,7 +62,35 @@ const initializeWidget = async (backendUrl, container) => {
   }
 }
 
+/**
+ * Widgets should get initialized only once.
+ * Sets global variable to block other embedded bootstrap scripts from execution.
+ * Only the first bootstrap script gets executed.
```

Comment:
what happens if multiple widgets are embedded like this in the same page:

```
<div data-tocco-widget-key="1"></div>
<script src=".../bootstrap.js"></script>

foo bar...

<div data-tocco-widget-key="2"></div>
<script src=".../bootstrap.js"></script>
```

Is the second widget rendered as well or is it ignored because the second `<div>` is not yet in the DOM when the first bootstrap script is executed? I think we should wait with the bootstrapping until the DOM is fully loaded?